ref/Castle.Core-net6.0.cs

```diff
@@ -205,6 +205,7 @@ public class DictionaryAdapterInstance
         public System.Collections.Generic.IDictionary<string, Castle.Components.DictionaryAdapter.PropertyDescriptor> Properties { get; }
         public void AddCopyStrategy(Castle.Components.DictionaryAdapter.IDictionaryCopyStrategy copyStrategy) { }
     }
+    [System.Diagnostics.DebuggerDisplay("Type: {Type.FullName,nq}")]
```

Comment:
(Seeing this change while reviewing the PR commit by commit:)

This is strange. Updating stuff in the test project shouldn't change the `ref/` file for the main library. Should this change (and the other ones in the same file) have been included in an earlier commit?

Reply:
I'm struggling to recreate this now (i.e. the version of the file without this addition - I remember being confused by it at the time, but I can't remember the cause. Quite likely it is related to a previous commit, and I didn't rerun all the tests at the time, so found this change at a later point.

Reply:
The class has the attribute, so resolving this comment:

https://github.com/castleproject/Core/blob/38298f3ce58902a354412ad6be136c367f12a95e/src/Castle.Core/Components.DictionaryAdapter/DictionaryAdapterMeta.cs#L23-L24